docs/environments/testnet.md

```diff
@@ -59,6 +51,13 @@ listed at the bottom of that page when you're done playing with the testnet.
 
 ---
 
+## Testnet Block Explorer
+
+A testnet instance of [dcrdata](https://github.com/decred/dcrdata) is available for public
+use at <https://testnet.dcrdata.org/>.
+
+---
+
 ## Testnet Voting Service Providers (VSP)
```

Comment:
Maybe provide a little more info on the list of testnet VSPs if possible? E.g. "Below is a list of VSPs that support testnet"

https://test-dcrpool.dittrex.com/ is returning a 502 error...

Reply:
Added a caveat:

> As these are test environments, they may experience periods of instability or downtime.